docs/preview/02-Guidance/Service-to-service Correlation/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Service-to-service Correlation?"
+layout: default
+---
+
+# Service-to-service correlation
+The concept of service-to-service correlation is big and complex and spans multiple Arcus libraries. This user-guide will walk through all the available Arcus features that work together to facilitate service-to-service correlation in your application.
+
+## What is service-to-service correlation?
+Service-to-service correlation is a way to describe a relationship between different components where each 'component' represents a separated application. This could be, for example, an API call that results in an Azure Service Bus message that will be picked up by a message pump, or Azure Service Bus Azure Function that calls an additional API after receiving a message from a queue.
```

Comment:
```suggestion
Service-to-service correlation is a way to describe a relationship between different components where each 'component' represents a separated application. This could be, for example, an API call that results in an Azure Service Bus message that will be picked up by a message pump, or an Azure Function that calls an additional API after receiving a message from a queue.
```